components/application-api/base/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- https://github.com/johnmcollier/application-api/config/kcp?ref=addAPIs
```

Comment:
needs to be updated after https://github.com/redhat-appstudio/application-api/pull/1 is merged

Reply:
Updated